### ISSUE TITLE
Why is this happening? TypeError: Temperature is not a number: N/A

### ISSUE THREAD

**Issue by Opontios** [georgezhao2010/midea_ac_lan#69]
Can anyone explain why is this happening?
```
Logger: homeassistant.components.climate
Source: helpers/temperature.py:24
Integration: Климат (documentation, issues)
First occurred: 17:47:13 (2 occurrences)
Last logged: 17:47:13
Error adding entities for domain climate with platform midea_ac_lan
Error while setting up midea_ac_lan platform for climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 428, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 673, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 776, in add_to_platform_finish
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 532, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 572, in _async_write_ha_state
    attr.update(self.state_attributes or {})
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 270, in state_attributes
    ATTR_CURRENT_TEMPERATURE: show_temp(
  File "/usr/src/homeassistant/homeassistant/helpers/temperature.py", line 24, in display_temp
    raise TypeError(f"Temperature is not a number: {temperature}")
TypeError: Temperature is not a number: N/A
```

**Comment by georgezhao2010** [comment]
> Can anyone explain why is this happening?
> 
> ```
> Logger: homeassistant.components.climate
> Source: helpers/temperature.py:24
> Integration: Климат (documentation, issues)
> First occurred: 17:47:13 (2 occurrences)
> Last logged: 17:47:13
> Error adding entities for domain climate with platform midea_ac_lan
> Error while setting up midea_ac_lan platform for climate
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 428, in async_add_entities
>     await asyncio.gather(*tasks)
>   File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 673, in _async_add_entity
>     await entity.add_to_platform_finish()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 776, in add_to_platform_finish
>     self.async_write_ha_state()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 532, in async_write_ha_state
>     self._async_write_ha_state()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 572, in _async_write_ha_state
>     attr.update(self.state_attributes or {})
>   File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 270, in state_attributes
>     ATTR_CURRENT_TEMPERATURE: show_temp(
>   File "/usr/src/homeassistant/homeassistant/helpers/temperature.py", line 24, in display_temp
>     raise TypeError(f"Temperature is not a number: {temperature}")
> TypeError: Temperature is not a number: N/A
> ```

Modify the file `midea_ac_lan/midea/devices/ac/devices` line 93,  replace `DeviceAttributes.indoor_temperature:  "N/A"` with `DeviceAttributes.indoor_temperature:  0.0`

**Comment by rokam** [comment]
I suggest to use `None` instead of `0.0`.
If you start using indoor and outdoor temperatures as long term statistics, 0.0 should compromise min and mean statistics.

**Comment by Opontios** [comment]
Putting 0.0 really helped. Putting "None" didn't. The strange thing is that I've been using this integration for quite some time, and following the latest updates, but today I decided to reconfigure the integration, as it had non user friendly entity_ids end after re-configuring i got this error message. Thanks for the help.

**Comment by rokam** [comment]
None for me suppress the error

**Comment by SkinnyT81** [comment]
Changing to 0.0 worked for me as well.  Thanks!

**Comment by georgezhao2010** [comment]
fixed in [v0.3.7](https://github.com/georgezhao2010/midea_ac_lan/releases/tag/v0.3.7)
